pkg/cmd/update/update.go

```diff
@@ -18,7 +19,10 @@ var UpdateCmd = &cobra.Command{
 	Long:  ``,
 	Run: func(cmd *cobra.Command, args []string) {
 		updateVersion := args[0]
-		var homeDir = "/usr/local/bin/"
+		homeDir, err := homedir.Dir()
+		if err != nil {
+			utils.PrintError(err)
+		}
```

Comment:
Hey @SarthakJain26, Thanks for your meaningful suggestion!, I have implemented the same here.